packages/web/src/app/components/canvas/TileSVG.tsx

```diff
@@ -63,4 +63,10 @@ const TileSVG = ({
   );
 };
 
-export default TileSVG;
+export default React.memo(
+  TileSVG,
+  (prev, next) =>
+    prev.viewbox == next.viewbox &&
+    prev.svgWidth == next.svgWidth &&
+    prev.svgHeight == next.svgHeight
+);
```

Comment:
yup

Reply:
I didn't spot this earlier it's good practice to use `===` instead of `==` in ~almost all circumstances:
https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

The only instance in which `==` is used is `value == null` or `value != null` where your intent is to check for `null` or `undefined` (i.e. a shorthand for `value === null || value === undefined`).